src/test/java/jenkins/branch/OrganizationFolderTest.java

```diff
@@ -127,6 +134,9 @@ public void emptyViewEquality() throws Exception {
     @Issue("JENKINS-34246")
     @Test
     public void deletedMarker() throws Exception {
+        assumeThat("TODO fails if jth.jenkins-war.path includes WorkflowMultiBranchProjectFactory since SingleSCMSource ignores SCMSourceCriteria",
```

Comment:
TODO?  What will you be doing here?

Reply:
Potentially rewriting the test to do something less brittle, like using `MockSCM` as in newer tests in this class.